### PR TITLE
Pr/various3

### PR DESCRIPTION
**Pull request by uli42**
Some small commits

-

### CODE REVIEW

**Comment by sunweaver** [comment]
please rebase pr/various3 against 3.6.x, thanks.

**Comment by uli42** [comment]
done

File: nx-X11/programs/Xserver/Xext/xf86bigfont.c [hunk 1]
```diff
@@ -50,7 +50,7 @@
 #ifdef SVR4
 #include <sys/sysmacros.h>
 #endif
-#if defined(ISC) || defined(__CYGWIN__) || defined(__SCO__)
+#if defined(__CYGWIN__) || defined(__SCO__)
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/Xext/xf86bigfont.c:53]: unifdef ISC, this is...

Comment (sunweaver) [reply nx-X11/programs/Xserver/Xext/xf86bigfont.c:53]: and while unifdef'in WIN32, we should do that in the Imakefiles, too?

**Review by sunweaver — APPROVED** [review]
LGTM (I will fix comments myself)

**Pull request merged**